.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,102 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+``` js
+/^(revert: )?(feat|fix|docs|style|refactor|test|ci|chore)(\(.+\))?: .{1,50}/
+```
+
+#### Types
+* `feat` - feature. Something noticeable to end user. When feature takes several days of work - split it into several smaller features.
```

Comment:
New functionality. Consider splitting into smaller chunks if takes several days of work.